tests/test_utils/emg_fixtures.py

```diff
@@ -622,17 +624,50 @@ def assemblies(study, runs, samples, experiment_type_assembly):
         accession="ERZ9999",
         study=study,
         experiment_type=experiment_type_assembly,
-        _quantity=10))
+        _quantity=1))
```

Comment:
`1` instead of `10`?

Reply:
Yeh. I believe this was a typo... the comment above this is `# one with a fixed ERZ9999 to be used with the legacy mapping` and indeed it only makes sense to make 1 with a hardcoded accession. Doesn't seem to violate any DB uniqueness checks (probably should!), but I noticed it was wrong when doing a `.get` for this supposedly unique accession.